MSAL/src/native_auth/controllers/reset_password/MSALNativeAuthResetPasswordControlling.swift

```diff
@@ -26,15 +26,20 @@
 
 protocol MSALNativeAuthResetPasswordControlling: AnyObject {
 
-    func resetPassword(parameters: MSALNativeAuthResetPasswordStartRequestProviderParameters) async -> ResetPasswordStartResult
+    typealias ResetPasswordStartControllerResponse = MSALNativeAuthControllerTelemetryWrapper<ResetPasswordStartResult>
+    typealias ResetPasswordResendCodeControllerResponse = MSALNativeAuthControllerTelemetryWrapper<ResetPasswordResendCodeResult>
+    typealias ResetPasswordSubmitCodeControllerResponse = MSALNativeAuthControllerTelemetryWrapper<ResetPasswordVerifyCodeResult>
+    typealias ResetPasswordSubmitPasswordControllerResponse = MSALNativeAuthControllerTelemetryWrapper<ResetPasswordRequiredResult>
```

Comment:
Should this be `MSALNativeAuthControllerTelemetryWrapper<ResetPasswordSubmitPasswordResult>` to keep in sync with the rest?

Reply:
Yes, it makes sense. Updated 👍